spec/system/composer/category_templates_spec.rb

```diff
@@ -24,6 +24,14 @@
   fab!(:form_template_4) do
     Fabricate(:form_template, name: "Biography", template: "- type: textarea")
   end
+  fab!(:form_template_5) do
+    Fabricate(
+      :form_template,
+      name: "Medication",
+      template:
+        "- type: input\n  attributes:\n    label: \"What is your name?\"\n    placeholder: \"John Smith\"\n  validations:\n    required: false\n- type: upload\n  attributes:\n    file_types: \".jpg, .png\"\n    allow_multiple: true\n    label: \"Upload your prescription\"\n  validations:\n    required: true\n- type: upload\n  attributes:\n    file_types: \".jpg, .png\"\n    allow_multiple: false\n    label: \"Any additional docs\"\n  validations:\n    required: false",
```

Comment:
I find this very hard to read. Perhaps we can use the heredoc syntax to improve readability here?